app/src/main/res/layout-sw600dp/administrator_controls_account_actions_view.xml

```diff
@@ -18,14 +18,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:background="@color/oppiaGreyBackground"
-      android:fontFamily="sans-serif-medium"
+      style="@style/Subtitle2"
```

Comment:
That should be `style="@style/Caption` because the TextView had:  
```
android:fontFamily="sans-serif-medium"
android:textSize="14sp"
```
And these attributes can be found in `Caption`:  
```
<style name="Caption" parent="TextView.Common1">
    <item name="android:fontFamily">sans-serif-medium</item>
    <item name="android:textSize">14sp</item>
    <item name="fontFamily">sans-serif-medium</item>
  </style>
```
not in `Subtitle2`:
```
<style name="Subtitle2" parent="TextView.Common1">
    <item name="android:fontFamily">sans-serif</item>
    <item name="android:textSize">14sp</item>
    <item name="fontFamily">sans-serif</item>
  </style>
```

Reply:
@MohamedMedhat1998 You can suggest the author in such PRs to remove the `layout_height` and `layout_width` too, depending upon the code.

Reply:
@rt4914 but from where will the view get its `layout_height` and `layout_width`?

Reply:
`Common1` already covers it. 
https://github.com/oppia/oppia-android/blob/cb27a1c68ecb77de2b21b82b7c4535be411bb6d3/app/src/main/res/values/styles.xml#L242

Reply:
> `Common1` already covers it.
> 
> https://github.com/oppia/oppia-android/blob/cb27a1c68ecb77de2b21b82b7c4535be411bb6d3/app/src/main/res/values/styles.xml#L242

Ok, thanks!

Reply:
> `Common1` already covers it.
> https://github.com/oppia/oppia-android/blob/cb27a1c68ecb77de2b21b82b7c4535be411bb6d3/app/src/main/res/values/styles.xml#L242

No it doesn't cover layout here cos the layout_width is macth_parent:
 `android:layout_width="match_parent"
  android:layout_height="wrap_content"`
 but the one defined in style.xml is wrap_content:
`<style name="TextView.Common1" parent="Widget.AppCompat.TextView">
<item name="android:layout_width">wrap_content</item>
<item name="android:layout_height">wrap_content</item>`

Reply:
@secjedi you are right here, but I think Rajat was speaking in general, not in this specific case.